src/board/board.spec.js

```diff
@@ -0,0 +1,27 @@
+import Board from './board';
+
+describe('Updates board', () => {
```

Comment:
Two things:

1. There is still a bit of a mismatch between the test descriptions and what the test is doing.

For example, the test description says `"Updates board"` with the verb being "update." The action in each of the tests is `mark`. So is it "updating the board" or "marking the board" ?

2. I feel like the description would flow a bit better if it read `"Updating the board"` instead of `"Updates board"`